html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Kickbox.pm

```diff
@@ -0,0 +1,65 @@
+package pfappserver::Form::Config::Authentication::Source::Kickbox;
+
+=head1 NAME
+
+pfappserver::Form::Config::Authentication::Source::Kickbox add documentation
+
+=cut
+
+=head1 DESCRIPTION
+
+pfappserver::Form::Config::Authentication::Source::Kickbox
```

Comment:
What a great name ! But hey, that's the description field ! How about a description instead ? ;)